numpy/core/_ufunc_config.py

```diff
@@ -154,7 +154,7 @@ def geterr():
     --------
     >>> np.geterr()
     {'divide': 'warn', 'over': 'warn', 'under': 'ignore', 'invalid': 'warn'}
-    >>> np.arange(3.) / np.arange(3.)  # doctest: +SHOW_WARNINGS
+    >>> np.arange(3.) / np.arange(3.)
     array([nan,  1.,  1.])
     RuntimeWarning: invalid value encountered in divide
```

Comment:
I believe the exception needs to be removed here in here.

```suggestion
```

Reply:
I think the warning is correct, but maybe warnings are set up differently in the two runs.  With pytest, we have the problem that warnings are set to errors (when they should probably not be for the doctest part), and maybe the current checks don't handle them at all?

In any case, I tend towards the +SKIP.  If we forget about it... this function should maybe be hidden away from public anyway...